programs/intent-transfer/src/bridge/processor/bridge_ntt_tokens.rs

```diff
@@ -404,3 +399,103 @@ fn parse_recipient_address(address_str: &str) -> Result<[u8; 32]> {
 
     Ok(result)
 }
+
+/// Computes the relay ntt args to pass to the NTT with executor CPI.
+fn compute_relay_ntt_args(
+    to_chain_id_wormhole: u16,
+    signed_quote_bytes_serialized: Vec<u8>,
+    pay_destination_rent: bool,
+) -> Result<RelayNttMessageArgs> {
+    let (msg_value, gas_limit) = if to_chain_id_wormhole == 1 {
+        compute_msg_value_and_gas_limit_solana(pay_destination_rent)
+    } else {
+        return Err(IntentTransferError::UnsupportedToChainId.into());
+    };
+
+    // constructed in line with the gas instruction format: https://github.com/wormholelabs-xyz/example-messaging-executor?tab=readme-ov-file#relay-instructions
+    let relay_instructions = [
+        [1u8].as_slice(),
+        gas_limit.to_be_bytes().as_slice(),
+        msg_value.to_be_bytes().as_slice(),
+    ]
+    .concat();
+
+    let signed_quote_bytes = SignedQuoteBytes::try_from_slice(&signed_quote_bytes_serialized)
+        .map_err(|_| IntentTransferError::InvalidNttSignedQuote)?;
+    let exec_amount = compute_exec_amount(
+        to_chain_id_wormhole,
+        signed_quote_bytes,
+        gas_limit,
+        msg_value,
+    )?;
+
+    Ok(RelayNttMessageArgs {
+        recipient_chain: to_chain_id_wormhole,
+        exec_amount,
+        signed_quote_bytes: signed_quote_bytes_serialized,
+        relay_instructions,
+    })
+}
+
+pub const LAMPORTS_PER_SIGNATURE: u128 = 5000;
+pub const RENT_TOKEN_ACCOUNT_SOLANA: u128 = 2_039_280; // equals 0.00203928 SOL
+
+/// Based on the logic encoded in https://github.com/wormhole-foundation/native-token-transfers/blob/20fc162f4a37391694dfb0e31afedf72549ea477/solana/ts/sdk/nttWithExecutor.ts#L304-L344.
+fn compute_msg_value_and_gas_limit_solana(pay_destination_rent: bool) -> (u128, u128) {
```

Comment:
what about returning (u64,u64) (since these numbers are small) here so you don't have to convert it later in `compute_exec_amount`